macros/cloud_safe_print_table.sql

```diff
@@ -0,0 +1,29 @@
+{% macro is_cloud() %}
+    {# In the dbt Cloud context, `DBT_ENV` will always resolve to "prod" #}
+    {% set is_cloud = env_var("DBT_ENV", "core") == "prod" %}
+    {{ return(is_cloud) }}
+{% endmacro %}
+
+{% macro cloud_safe_print_table(audit_results) %}
+{% if execute %}
+    {% if is_cloud() %}
+        {% set header = [] %}
+        {% for i in range(0,audit_results.column_names|length) %}
+            {{ header.append(audit_results.column_names[i]) }}
+        {% endfor %}
+        {% do log('| ' ~ header|join(' | ') ~ ' |', info=True) %}
+        {# do log(audit_results.column_names, info=True) #}
+        {% for row in audit_results.rows %}
+            {% set clean_row = [''] %}
+            {% for val in row.values() %}
+                {{ clean_row.append(val) }}
+            {% endfor %}
+            {% do log(clean_row|join(' | ') ~ ' |', info=True) %}
+            {# do log(row.values(), info=True) #}
+        {% endfor %}
```

Comment:
Cool @davesgonechina ! 😎 

Absolutely ideal would be to handle padding too, like this:
https://stackoverflow.com/questions/45698629/jinja2-padding-and-aligning-strings

We're all out of the office until January 3rd, so I'll take a look at this more after that.

Reply:
Same - also, looks like after I opened this PR a new macro https://github.com/dbt-labs/dbt-audit-helper#compare_all_columns-source was added, and it looks like perhaps it does something very similar to this PR? Will take another look.